src/ir_Mitsubishi.cpp

```diff
@@ -1186,3 +1198,449 @@ String IRMitsubishi136::toString(void) {
   result += addBoolToString(getQuiet(), F("Quiet"));
   return result;
 }
+
+
+#if SEND_MITSUBISHI112
+// Send a Mitsubishi112 A/C message.
+//
+// Args:
+//   data: An array of bytes containing the IR command.
+//   nbytes: Nr. of bytes of data in the array. (>=kMitsubishi112StateLength)
+//   repeat: Nr. of times the message is to be repeated.
+//          (Default = kMitsubishi112MinRepeat).
+//
+// Status: ALPHA / Probably working. Needs to be tested against a real device.
```

Comment:
Hey, you said you had it working on your device?! :) You should mark this as Beta at the least, if not Stable. ;-)